jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/constants/PacmanRuleConstants.java

```diff
@@ -690,4 +690,8 @@ private PacmanRuleConstants() {
     public static final String PROJECT_ID= "projectId";
 
     public static final String PROJECT_NAME= "projectName";
+
+    public static final String CLOUD_WATCH_LOGS_ARN= "cloudWatchLogsLogGroupArn";
+
+    public static final String LATEST_CLOUD_WATCH_DELIVERY_TIME="latestCloudWatchLogsDeliveryTime";

```

Comment:
Do you see any usage cases for these two in future?

Reply:
Used in EnableCloudTrailCloudwatchLogsIntegration.java file